cfme/tests/cloud_infra_common/test_html5_vm_console_ssui.py

```diff
@@ -0,0 +1,147 @@
+"""Test for HTML5 Remote Consoles of VMware/RHEV/RHOSP Providers."""
+import pytest
+from wait_for import wait_for
+
+from cfme import test_requirements
+from cfme.cloud.provider.openstack import OpenStackProvider
+from cfme.infrastructure.provider.rhevm import RHEVMProvider
+from cfme.infrastructure.provider.virtualcenter import VMwareProvider
+from cfme.services.myservice import MyService
+from cfme.utils.appliance import ViaSSUI
+
+
+@pytest.mark.parametrize('context', [ViaSSUI])
+@test_requirements.html5
+@pytest.mark.provider([OpenStackProvider, VMwareProvider, RHEVMProvider])
+@pytest.mark.uncollectif(lambda provider, appliance:
+    (provider.one_of(VMwareProvider) and provider.version >= 6.5 and appliance.version < '5.11') or
+    (provider.one_of(RHEVMProvider) and provider.version < 4.3),
+    reason='VNC Not supported for CFME 5.10 with VMware 6.5 onward')
+@pytest.mark.parametrize('order_service', [{'console_test': True}], indirect=True)
+def test_vm_console_ssui(request, appliance, setup_provider, provider, context,
+     configure_console_webmks, configure_console_vnc, order_service, take_screenshot,
+     configure_websocket, console_template):
+    """Test Myservice VM Console in SSUI.
+
+    Metadata:
+        test_flag: ssui
+
+    Polarion:
+        assignee: apagac
```

Comment:
Did you ever hear from Sudhir about this? I don't think we should leave tests assigne to `apagac`

Reply:
Nope :(

Reply:
I'm okay with it, easy to bulk re-assign.